velox/functions/lib/Re2Functions.cpp

```diff
@@ -15,6 +15,7 @@
  */
 #include "velox/functions/lib/Re2Functions.h"
 
+#include <expression/EvalCtx.h>
```

Comment:
Why is this needed?

Reply:
My c/c++ code insights plug-in tends to add headers it believes are needed. I removed it and ran tests which all passed.